addons/binding/org.openhab.binding.vera/src/main/java/org/openhab/binding/vera/handler/VeraDeviceHandler.java

```diff
@@ -0,0 +1,468 @@
+                    }
+                }
+            } else {
+                logger.warn("Device {} not loaded", deviceId);
+            }
+        }
+    }
+
+    protected synchronized void addDeviceAsChannel(Device device, boolean homekitIntegration) {
+        if (device != null) {
+            logger.debug("Add device as channel: {}", device.getName());
+
+            HashMap<String, String> properties = new HashMap<>();
+            properties.put("deviceId", device.getId());
+
+            String id = null;
+            String acceptedItemType = "";
+            String tag = null;
+
+            int subcategory = Integer.parseInt(device.getSubcategory());
+            switch (device.getCategoryType()) {
+                case Controller:
+                case Interface:
+                    break;
+                case DimmableLight:
+                    switch (subcategory) {
+                        case 1:
+                        case 2:
+                        case 3:
+                            id = SWITCH_MULTILEVEL_CHANNEL;
+                            acceptedItemType = "Dimmer";
+                            tag = "Lighting";
+                            break;
+                        case 4:
+                            id = SWITCH_COLOR_CHANNEL;
+                            acceptedItemType = "Color";
+                            tag = "Lighting";
+                            break;
+                    }
+                    break;
+                case Switch:
+                    id = SWITCH_BINARY_CHANNEL;
+                    acceptedItemType = "Switch";
+                    tag = "Switchable";
+                    break;
+                case SecuritySensor:
+                    switch (subcategory) {
+                        case 1:
+                            id = SENSOR_DOOR_WINDOW_CHANNEL;
+                            acceptedItemType = "Contact";
+                            break;
+                        case 2:
+                            id = SENSOR_FLOOD_CHANNEL;
+                            acceptedItemType = "Switch";
+                        case 3:
+                            id = SENSOR_MOTION_CHANNEL;
+                            acceptedItemType = "Switch";
+                            break;
+                        case 4:
+                            id = SENSOR_SMOKE_CHANNEL;
+                            acceptedItemType = "Switch";
+                            break;
+                        case 5:
+                            id = SENSOR_CO_CHANNEL;
+                            acceptedItemType = "Switch";
+                            break;
+                        case 6:
+                            id = SENSOR_BINARY_CHANNEL;
+                            acceptedItemType = "Switch";
+                            break;
+                    }
+                    break;
+                case HVAC: // TODO
+                    logger.warn("TODO: {}, {}", device, device.getCategoryType());

```

Comment:
Can you change it to `debug` instead? I wouldn't like a log file full of warnings just because devices aren't yet supported by a binding. It would also help to make a note about this in the docs.